src/domain/products/details/variants/index.js

```diff
@@ -62,6 +62,15 @@ const Variants = ({
     },
   ]
 
+  useEffect(() => {
+    if (
+      variants &&
+      variants[0] &&
+      variants[variants.length - 1].id === undefined
+    )
+      setNewVariant(variants[variants.length - 1])
+  }, [variants])
+
```

Comment:
Whenever we want to create a new variant we add a blank one to the list of variants, and then hereafter we must select it for opening - which is why we do this